pkg/api/types.go

```diff
@@ -5,6 +5,10 @@ import docker "github.com/fsouza/go-dockerclient"
 const (
 	DefaultNamespace    = "io.openshift.s2i."
 	KubernetesNamespace = "io.k8s."
+	DefaultIgnoreFile   = ".s2iignore"
+	DefaultUploadSubdir = "upload"
+	DefaultSrcSubdir = "src"
```

Comment:
i've never bothered to learn....  does windows have an equivalent to osx boot2docker?  and when you're using it, is it transparent (eg you get a docker daemon in your host that proxies into the VM)?

if so then we would need to worry about this.

looks like yes: https://docs.docker.com/installation/windows/

but i have no idea if the docker client libraries we are using would work in that environment.  still, probably better to not do it so we don't have a problem later.